unime/src/lib/connections/sorting/SortingSheetButton.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import { createEventDispatcher, SvelteComponent } from 'svelte';
+
+  import LL from '$i18n/i18n-svelte';
+  import type { SvelteHTMLElements } from 'svelte/elements';
+
+  import type { CredentialSortMethod } from '@bindings/CredentialSortMethod';
+
+  export let method: CredentialSortMethod;
+  export let icon: typeof SvelteComponent<SvelteHTMLElements['svg']>;
+  export let label: string;
+  export let active = false;
+  export let reversed: boolean;
+
+  // Compute sort order based on label and reversal
+  let sortOrder: string;
+  $: {
+    if (method === 'name_az') {
+      sortOrder = reversed ? $LL.SORT.ORDER.Z_A() : $LL.SORT.ORDER.A_Z();
+    } else if (method === 'issue_date_new_old') {
+      sortOrder = reversed ? $LL.SORT.ORDER.NEWEST() : $LL.SORT.ORDER.OLDEST();
+    } else if (method === 'added_date_new_old') {
+      sortOrder = reversed ? $LL.SORT.ORDER.NEWEST() : $LL.SORT.ORDER.OLDEST();
+    } else {
+      sortOrder = ''; // Default to empty string if label doesn't match
+    }
+  }
+
+  // Dispatch click event
```

Comment:
I don't think that comment is necessary since the code below explains itself.

Reply:
Maybe something like: // Dispatch component event based on DOM event.